build/installer.nsh

```diff
@@ -19,6 +60,153 @@ Function uninstallSymphony
+    Exch 2
+    Exch $2 ;string to write
+    Exch 2
+    Push $3
+    Push $4
+	
+
+    FileOpen $4 $0 r
+	GetTempFileName $R0 $PROGRAMFILES64\Symphony\Symphony\config
+	FileOpen $R4 $R0 w
+	
+	#FileOpen $8 "E:\Projects\NSIS\$1_0.txt" w This one is for debugging purpose if you need, open it
+	#CopyFiles /SILENT $0 "E:\Projects\NSIS\$1.txt"
+
+	loopFind:
+		FileRead $4 $R1
+		${If} $R1 == ""
+			#FileWrite $8 "At Error"
+			#FileWrite $8 "$R1$\r$\n"
+			Goto done
+		${Else}
+			${StrContains} $R5 "$\"url$\":" $R1
+			#FileWrite $8 "$R1$\r$\n"
+			#FileWrite $8 "At loop find"
+
+			StrCmp $R5 "" notfound
+				#FileWrite $8 "$R5$\r$\n"
+				#FileWrite $8 "At URL found"
+				FileWrite $R4 "$R1"
+				FileWrite $R4 "$2$\r$\n"
+				Goto loopFind
+			notfound:
+				${StrContains} $R6 $1 $R1
+				#FileWrite $8 "At not found"
+				#FileWrite $8 "$R6$\r$\n"
+				StrCmp $R6 "" notmatchkey
+					#FileWrite $8 "At key found"
+					#FileWrite $8 "$R6$\r$\n"
+					Goto loopFind
+				notmatchkey:
+					#FileWrite $8 "At key not found"
+					#FileWrite $8 "$R1$\r$\n"
+					FileWrite $R4 $R1
+					Goto loopFind
+		${EndIf}
+		done:
+			FileClose $R4
+			FileClose $4
+			#FileWrite $8 "At done"
+			#FileWrite $8 "$4$\r$\n"
+			#FileWrite $8 "At done"
+			#FileWrite $8 "$R4$\r$\n"
+			CopyFiles /SILENT $R0 $0
+			Delete $R0
+			pop $0
+			pop $1
+			pop $2
+			pop $3
+			pop $4
+			pop $5
+			pop $6
+			pop $R0
+			pop $R1
+			pop $R4
+			pop $R5
+			pop $R6
+FunctionEnd
+
+
+Function SearchAndReplace
+	StrCpy $3 "$PROGRAMFILES64\Symphony\Symphony\config\Symphony.config"
+    pop $0
+	pop $1
+    StrCpy $4 "$\t$\"$0$\": $1,"
+	
+	#FileOpen $8 "E:\Projects\NSIS\text.txt" w
+	#FileWrite $8 "$0$\r$\n"
+	#FileWrite $8 "$1$\r$\n"
+	#FileWrite $8 "$4$\r$\n"
+	#FileClose $8
+
+	#LogText $0
+	#DetailPrint $0
+	#DetailPrint $1
+	#DetailPrint $4
+
+	# Find the term in the config
+
+	IfFileExists $3 0 +2
+        Push $4
+        Push $0
+        Push $3
+        Call WriteFindAt
+	Pop $0
+	Pop $1
+	Pop $2
+	Pop $3
+	Pop $4
+
+FunctionEnd
+
+# isPodUrlEditable
+Function updateConfigIsPodUrlEditable
+	push true
+	push isPodUrlEditable
+	call SearchAndReplace
+FunctionEnd
+
+# forceAutoUpdate
+Function updateConfigForceAutoUpdate
+	push false
+	push forceAutoUpdate
+	call SearchAndReplace
+FunctionEnd
+
+# enableBrowserLogin
+Function updateConfigEnableBrowserLogin
+	push false
+	push enableBrowserLogin
+	call SearchAndReplace
+FunctionEnd
+
+# browserLoginAutoConnect
+Function updateConfigBrowserLoginAutoConnect
+	push false
+	push browserLoginAutoConnect
+	call SearchAndReplace
+FunctionEnd
+
+# betaAutoUpdateChannelEnabled
+Function updateConfigBetaAutoUpdateChannelEnabled
+	push true
+	push betaAutoUpdateChannelEnabled
+	call SearchAndReplace
+FunctionEnd
+
+# latestAutoUpdateChannelEnabled
+Function updateConfigLatestAutoUpdateChannelEnabled
+	push true
+	push latestAutoUpdateChannelEnabled
+	call SearchAndReplace
+FunctionEnd
+
 !macro copySystemGlobalConfig
```

Comment:
This logic also needs to be applied for per user install

Reply:
I added the logic to per user